arlo-download.py

```diff
@@ -0,0 +1,48 @@
+from datetime import timedelta, date
+from Arlo import Arlo
+#import json
+import datetime
+
+USERNAME = 'user@example.com'
+PASSWORD = 'supersecretpassword'
+
+try:
+	# Instantiating the Arlo object automatically calls Login(), which returns an oAuth token that gets cached.
+	# Subsequent successful calls to login will update the oAuth token.
+	arlo = Arlo(USERNAME, PASSWORD)
+	# At this point you're logged into Arlo.
+
+	today = (date.today()-timedelta(days=0)).strftime("%Y%m%d")
+	seven_days_ago = (date.today()-timedelta(days=7)).strftime("%Y%m%d")
+
+	# Get all of the recordings for a date range.
+	library = arlo.GetLibrary(seven_days_ago, today)
+
+	# Iterate through the recordings in the library.
+	for recording in library:
+
+		# Get video as a chunked stream; this function returns a generator.
+		stream = arlo.StreamRecording(recording['presignedContentUrl'])
+		videofilename = datetime.datetime.fromtimestamp(int(recording['name'])//1000).strftime('%Y-%m-%d %H-%M-%S') + ' ' + recording['uniqueId']
```

Comment:
Same as in the README. Let's append the .mp4 to videofilename, so we can stop appending it all over the place below.